style.css

```diff
@@ -73,9 +76,93 @@ main {
 footer {
     text-align: center;
     grid-area: footer;
-    color: var(--color);
+    color: var(--color) ;
     background-color: var(--back-color);
     border-top-left-radius: 0.5rem;
     border-top-right-radius: 0.5rem;
 }
+/* 
```

Comment:
is this commented code needed?

Reply:
No, I forgot to delete them. I will delete